homeassistant/components/broadlink/const.py

```diff
@@ -6,6 +6,7 @@
 
 DOMAINS_AND_TYPES = {
     Platform.CLIMATE: {"HYS"},
+    Platform.TIME: {"HYS"},
```

Comment:
Let's keep this alphabetical

Reply:
What do we have Ruff for? 😅

Reply:
Done

Reply:
For a lot of the checks. We also have a pylint rule that checks if `PLATFORMS` variables are alphabetical. But this is a dict, not a list, and it's called `DOMAINS_AND_TYPED` hence there is no check for this. But I personally do really like when these lists are neatly ordered to be consistent with the rest

Reply:
I added it as the datetime platform and later just renamed it. Shame on me^^